src/dialogs/examples/basic/O365ConnectorCardActionsDialogV2.ts

```diff
@@ -0,0 +1,119 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as teams from "botbuilder-teams";
+
+export class O365ConnectorCardActionsDialogV2 extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        // multiple choice examples
```

Comment:
Do we need to repeat **all** of these? The many lines of code could detract from what you're really trying to show (see my question above).

Reply:
Julian requested all three different layouts to integrate into Sample App.